R/dvhydrograph-render.R

```diff
@@ -287,6 +300,11 @@ getDVHydrographPlotConfig <- function(plotItem, plotItemName, yLabel="", minMaxE
                            lty=ifelse(plotItem[['newSet']] == "est", 1, 6), col=ifelse(plotItem[['newSet']] == "est", "orange", "red3")),
                       styles$est_lines)
     ),
+    estimated4Edges = list(
+      arrows = append(list(x0=plotItem[['time']], x1=plotItem[['time']], y0=plotItem[['y0']], y1=plotItem[['y1']], 
+                           lty=ifelse(plotItem[['newSet']] == "est", 1, 7), col=ifelse(plotItem[['newSet']] == "est", "black", "red3")),
```

Comment:
In the render function you should take that ifelse and replace `styles$est_lines` with it. The last 3 parameters in the styles entries come from styles$est_lines so that can be removed completely from the styles and its entry in the render function can be replaced with the ifelse